ui_framework/components/header_bar/header_bar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const KuiHeaderBar = ({ children, className, ...rest }) => {
+  const classes = classNames('kuiHeaderBar', className);
+  return (<div
+            className={classes}
+            {...rest}
+          >
+            {children}
+          </div>);
```

Comment:
Could you adjust this indentation https://github.com/elastic/kibana/pull/12277#discussion_r121467836 ?